js/button.js

```diff
@@ -1,227 +0,0 @@
-(function(){
-
-
-	function createFrame(){
-		markletFrame = document.createElement("iframe");
-		markletFrame.name = markletFrame.id = "arena_frame";
-		markletFrame.src = getURL({
-			url: window.location.href,
-			title: window.document.title
-		});
-		markletFrame.style.display = "none";
-		document.body.appendChild(markletFrame);
-	}
-
-
-	function createTarget(){
-		markletDiv = document.createElement("div");
-		markletDiv.id = "arena_div";
-		document.body.appendChild(markletDiv);
-	}
-
-
-	function createStyle(){
-		var a = chrome.extension.getURL("css/style.css");
-		var linkOpen = '<link type="text/css" rel="stylesheet" href="';
-		var linkClose = '">';
-		var headHTML = document.getElementsByTagName('head')[0].innerHTML;
-		headHTML += linkOpen.concat(a, linkClose);
-		document.getElementsByTagName('head')[0].innerHTML = headHTML;
-	}
-
-
-
-	function closeBookmarklet() {
-		if (markletFrame) document.body.removeChild(markletFrame);
-		if (markletDiv) document.body.removeNode(markletDiv);
-		if (markletStyle) document.body.removeNode(markletStyle);
```

Comment:
Believe we want all of these to be removeChild. Currently I get the following error:
```document.body.removeNode is not a function```